### PR TITLE
Create appbackplane.md

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@vamshikrishna-g 
@sanjaypujare 
Please verify and merge the PR.

File: docs/appbackplane.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Application Backplane
+
+Application backplane is a mechanism that enables communication between applications. Using this communication channel, the fraud detection outcomes can be shared across multiple applications so that fraud prevention in one application can signal fraud prevention in another application and further reduce the chance of fraud.
```
Comment (vamshikrishna-g) [inline docs/appbackplane.md]: Using this communication channel the fraud prevention or detection outcomes can be shared...

File: docs/appbackplane.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| **Operators** | **Description** |
+| --- | --- |
+| Kafka input operator | This operator emits the byte array from Kafka topic whose properties are set on the module. These properties must be the same ones set on the corresponding output module whose messages are subscribed. You can also indicate which offset to start reading messages from kafka. |
+| Avro deserialization | This operator does the deserialization based on  the schema set on the module. The schema must be the same one set on corresponding output module whose messages are subscribed. The deserialized class should be in classpath. This will be achieved by querying schema repository to include the jar of the class. |
+
+## Sample of Application Backplane Setup
+
+The following image depicts how **Account Takeover** application(ATO) uses the output module as a connector to send the data outcome to a configured Kafka topic.
+![](images/image1.png)
+This outcome can be used by **Omni Channel**** Fraud Prevention**application (**FP**), to obtain enhanced insights.
```
Comment (vamshikrishna-g) [inline docs/appbackplane.md]: Typo here, **Omni Channel Fraud Prevention** application

**Comment by lizygeogy** [comment]
@vamshikrishna-g, I have done the changes, please verify.

File: docs/appbackplane.md [hunk 3]
```diff
@@ -35,15 +35,15 @@ The input module has the following operators:
 
 The following image depicts how **Account Takeover** application(ATO) uses the output module as a connector to send the data outcome to a configured Kafka topic.
 ![](images/image1.png)
-This outcome can be used by **Omni Channel**** Fraud Prevention**application (**FP**), to obtain enhanced insights.
+This outcome can be used by **Omni Channel Fraud Prevention**application (**FP application**), to obtain enhanced insights.
```
Comment (vamshikrishna-g) [inline docs/appbackplane.md]: Missing empty space **Omni Channel Fraud Prevention** application

File: docs/appbackplane.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+The input module has the following operators:
+
+| **Operators** | **Description** |
+| --- | --- |
+| Kafka input operator | This operator emits the byte array from Kafka topic whose properties are set on the module. These properties must be the same ones set on the corresponding output module whose messages are subscribed. You can also indicate which offset to start reading messages from kafka. |
+| Avro deserialization | This operator does the deserialization based on  the schema set on the module. The schema must be the same one set on corresponding output module whose messages are subscribed. The deserialized class should be in classpath. This will be achieved by querying schema repository to include the jar of the class. |
+
+## Sample of Application Backplane Setup
+
+The following image depicts how **Account Takeover** application(ATO) uses the output module as a connector to send the data outcome to a configured Kafka topic.
```
Comment (vamshikrishna-g) [inline docs/appbackplane.md]: Missing space:  application(ATO)

**Pull request merged**